.github/workflows/deploy.yml

```diff
@@ -0,0 +1,56 @@
+name: de-project-vinson
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
is this going to be main later and you just do it this way now for testing purposes?

Reply:
According to Elliot - it should make the CI/CD deploy on every push, not just pushes to the main branch